golint warnings on consensus/util/messagefan.go

## Description

<!--- Describe your issue or user story in detail. -->

messagefan.go:33:1: comment on exported type Message should be of the form "Message ..." (with optional leading article)
messagefan.go:54:1: comment on exported method MessageFan.RegisterChannel should be of the form "RegisterChannel ..."
## Describe How to Reproduce

run golint

submitting a PR for this

--- a/consensus/util/messagefan.go
+++ b/consensus/util/messagefan.go
@@ -30,7 +30,7 @@ func init() {
 	logger = logging.MustGetLogger("consensus/util")
 }
 
-// Messages encapsulates an OpenchainMessage with sender information
+// Message encapsulates an OpenchainMessage with sender information
 type Message struct {
 	Msg    *pb.Message
 	Sender *pb.PeerID
@@ -51,7 +51,7 @@ func NewMessageFan() *MessageFan {
 	}
 }
 
-// registerChannel is intended to be invoked by Handler to add a channel to be fan-ed in
+// RegisterChannel is intended to be invoked by Handler to add a channel to be fan-ed in
 func (fan *MessageFan) RegisterChannel(sender *pb.PeerID, channel <-chan *Message) {
 	fan.lock.Lock()
 	defer fan.lock.Unlock()

